public/app/plugins/datasource/tempo/datasource.test.ts

```diff
@@ -391,6 +402,146 @@ describe('Tempo data source', () => {
     const lokiDS4 = ds4.getLokiSearchDS();
     expect(lokiDS4).toBe(undefined);
   });
+
+  it('should build expr correctly', () => {
```

Comment:
Nit: Maybe add APM table tests to a new describe since this is a large test file